server.ts

```diff
@@ -195,7 +196,14 @@ async function main() {
 
       let exitCode = 0;
       try {
-        const subscribers = await setupChainEventListeners(models, wss, SKIP_EVENT_CATCHUP);
+        // configure chain list from events
+        let chains: string[] | 'all' | 'none' = [ 'edgeware' ];
+        if (CHAIN_EVENTS === 'none' || CHAIN_EVENTS === 'all') {
+          chains = CHAIN_EVENTS;
```

Comment:
even if CHAIN_EVENTS=none all chains will be enabled?

Reply:
no -- this sets the chains variable to `none` which gets passed through and enables none of them